### PR TITLE
testing/fstrm: new aport

### PR DESCRIPTION
**Pull request by tcely**
https://github.com/farsightsec/fstrm
Frame Streams implementation in C

### CODE REVIEW

File: testing/fstrm/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+check() {
+	cd "$builddir"
+
+	make check
+}
+
+package() {
+	cd "$builddir"
+
+	make -j1 DESTDIR="$pkgdir" install
```
Comment (BernhardGruen) [inline testing/fstrm/APKBUILD]: Is this `-j1` really needed for the packaging step?

File: testing/fstrm/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Contributor:
```
Comment (BernhardGruen) [inline testing/fstrm/APKBUILD]: Please fill `Contributor` and `Maintainer` with your name and mail-address.

Comment (clandmeter) [reply testing/fstrm/APKBUILD]: Maintainer is only really needed when moving it to community or main.

**Comment by tcely** [comment]
Is there anything more to do before adding this?

**Comment by ncopa** [comment]
It would be nice if you could merge the commits. thanks!

**Comment by tcely** [comment]
@ncopa Squashed as requested.

**Pull request closed without merging**